packages/app/src/components/Settings/Settings.js

```diff
@@ -194,59 +214,114 @@ const Settings = (props) => {
-                name="closedOn"
-                onChange={handleChange}
-                type="date"
-              />
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
+              >
+                <DatePicker
+                  name="closedOn"
+                  value={newGrantCycle.closedOn}
+                  selected={newGrantCycle.closedOn}
+                  onChange={handleEndDateCreation}
+                  placeholderText="mm/dd/yyyy"
+                />
+              </div>
             </span>
           </div>
           <div className="settings__input-block">
             <p className="settings__input-label">Cycle Name:</p>
             <span className="settings__input">
-              <input
-                value={newGrantCycle.name}
-                name="name"
-                onChange={handleChange}
-                type="text"
-              />
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
+              >
+                <input
+                  value={newGrantCycle.name}
+                  name="name"
+                  onChange={handleChange}
+                  type="text"
+                />
+              </div>
             </span>
           </div>
         </div>
-        <button
-          ref={createButton}
-          disabled={disableButton}
-          onClick={handleCreate}
-          className="settings__button"
+        <div
           style={
-            disableButton
-              ? { color: 'gray', 'border-color': 'gray', 'font-weight': 'bold', 'opacity': '1' }
+            showEditModal
+              ? {
+                  display: 'none',
+                }
               : {
-                  color: 'var(--brand)',
-                  'background-color': '#fff',
-                  'border-color': 'var(--brand)',
+                  position: 'relative',
                 }
           }
         >
-          Create
-        </button>
+          <button
+            ref={createButton}
+            disabled={disableButton}
+            onClick={handleCreate}
+            className="settings__button"
+            style={
+              disableButton
+                ? {
+                    color: 'gray',
+                    borderColor: 'gray',
+                    fontWeight: 'bold',
+                    opacity: '1',
+                  }
+                : {
+                    color: 'var(--brand)',
+                    backgroundColor: '#fff',
+                    borderColor: 'var(--brand)',
+                  }
+            }
```

Comment:
Same here. I think it'd be better to have a `.button-disabled` somewhere with this CSS.

Reply:
Fixed everywhere!